crates/biome_js_parser/src/syntax/auxiliary.rs

```diff
@@ -96,6 +96,10 @@ pub(crate) fn parse_declaration_clause(p: &mut JsParser, stmt_start_pos: TextSiz
                     // }
                     parse_class_declaration(p, decorator_list, StatementContext::StatementList)
                 }
+                T![EOF] => {
+                    p.error(p.err_builder("Unexpected token after declare", p.cur_range()));
+                    Absent
+                }
```

Comment:
it seems we don't need this branch.
Because we call 'parse_declaration_clause' and the call returns 'Absent'.

Reply:
yes, i remove it~